beacon_node/network/src/beacon_processor/mod.rs

```diff
@@ -156,6 +157,10 @@ const MAX_BLOCKS_BY_RANGE_QUEUE_LEN: usize = 1_024;
 /// will be stored before we start dropping them.
 const MAX_BLOCKS_BY_ROOTS_QUEUE_LEN: usize = 1_024;
 
+/// The maximum number of queued `LightClientBootstrapRequest` objects received from the network RPC that
+/// will be stored before we start dropping them.
+const MAX_LIGHT_CLIENT_BOOTSTRAP_QUEUE_LEN: usize = 1_024;
```

Comment:
Just want to raise this length is probably higher than it needs to be. I can see it's on-par with blocks_by_root. 

The `blocks_by_root` has an RPC rate limit of 128 per peer per 10 seconds, whereas the light client bootstrap only allows 1 per peer per 10 seconds.

I think we'd rate limit before we'd hit this queue length. If this is supposed to be designed to be similar to blocks by root, might want to increase the RPC limit.

Reply:
Going to leave this as-is for now and we can revisit in future